.scrutinizer.yml

```diff
@@ -0,0 +1,23 @@
+tools:
+    php_sim: false
+    php_pdepend: true
+    php_analyzer: true
+
+build:
+    environment:
+        php:
+            version: 5.6.0
+    tests:
+        override:
+            -
+                command: 'vendor/bin/phpunit --verbose --configuration phpunit.xml --coverage-clover test/clover.xml'
```

Comment:
It\'s funny. I don\'t know how code coverage works for version_compare PHP_VERSION. It\'s clear Clover can not see unreachable lines so he will think these lines are not coveraged